src/NServiceBus.Persistence.AzureStorage.ComponentTests/Persisters/When_using_AzureStorageSagaGuard.cs

```diff
@@ -17,8 +16,7 @@ public void Should_not_allow_invalid_connection_string(string connectionString)
         [Test]
         public void Should_validate_all_default_settings_for_a_new_config()
         {
-            var config = new AzureSagaPersisterConfig();
-            Assert.AreEqual(AzureStorageSagaDefaults.CreateSchema, config.CreateSchema);
+            Assert.AreEqual(AzureStorageSagaDefaults.CreateSchema, AzureStorageSagaDefaults.CreateSchema);
```

Comment:
As far as I see it compares the default with itself. Shouldn't the test be removed then?